src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig

```diff
@@ -101,11 +101,12 @@
       {% endfor %}
       {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/add_product_row.html.twig' %}
       {% include '@PrestaShop/Admin/Sell/Order/Order/Blocks/View/edit_product_row.html.twig' %}
+
```

Comment:
empty line not needed

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/17660/commits/84aab034cb9252b408a502c914bae5ad65b1a994